Resources/Prototypes/Entities/Clothing/Neck/mantles.yml

```
@@ -62,4 +62,47 @@
   - type: Sprite
     sprite: Clothing/Neck/mantles/rdmantle.rsi
   - type: Clothing
-    sprite: Clothing/Neck/mantles/rdmantle.rsi    
+    sprite: Clothing/Neck/mantles/rdmantle.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckMantleColonel
+  suffix: DO NOT MAP
+  name: colonel's mantle
+  description: Gold alloy reinforced pauldrons signifying the rank of Colonel.
+  components:
+  - type: Tag
+    tags: [] # ignore "WhitelistChameleon" tag
+  - type: Sprite
+    sprite: Clothing/Neck/mantles/colonelmantle.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/mantles/colonelmantle.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckMantleCommodore
+  suffix: DO NOT MAP
+  name: commodore's mantle
+  description: A gold alloy reinforced pauldron signifying the rank of Commodore.
+  components:
+  - type: Tag
+    tags: [] # ignore "WhitelistChameleon" tag
+  - type: Sprite
+    sprite: Clothing/Neck/mantles/commodoremantle.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/mantles/commodoremantle.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckMantleMajor
+  suffix: DO NOT MAP
+  name: Major's mantle
+  description: A red padded pauldron signifying the rank of Major.
+  components:
+  - type: Tag
+    tags: [] # ignore "WhitelistChameleon" tag

```

im pretty sure something like this already overwrites `WhitelistChamaleon` (do this to everything else)
```suggestion
  - type: Tag # ignore "WhitelistChameleon" tag
```